docs/configuration/index.md

```diff
@@ -534,6 +534,16 @@ should trigger a reaction.
 should trigger a reaction.
 
 
+#### version_update_mask
+(*string*) This field requires a Python regex pattern. It's used with `re.match`.
```

Comment:
Maybe, we can mention a motivation first:
```suggestion
#### version_update_mask
(*string*) This field can be used to filter upstream versions when syncing the releases e.g. to avoid main version bump on the stable Fedora version.
The value requires a Python regex pattern and is used with `re.match` function.
```

Reply:
This was consistent with the above descriptions of  `upstream_tag_include` and `upstream_tag_exclude` and I think it is useful having this information quickly without need of reading too much 😅. But if you prefer I apply your changes.

Reply:
I don't have a strong option. I would expect people might not be sure just from the name what this is about...